doc/website/concepts/qos-policies.md

```diff
@@ -0,0 +1,30 @@
+# QoS policies
+
+Unlike other middlewares like for example [DDS used in ROS 2](https://docs.ros.org/en/galactic/Concepts/About-Quality-of-Service-Settings.html) iceoryx tries to keep things simple and offers only basic Quality of Service (QoS) settings. The QoS settings are called options and can be used to optimize and tailor the communication.
+
+## `PublisherOptions` and `SubscriberOptions`
+
+The three most important settings, which are also available in DDS, are:
+
+* `PublisherOptions::historyCapacity`: This will enable late-joining subscribers to read the n last samples
+* `SubscriberOptions::queueCapacity`: Size of the subscriber queue where samples are stored before they are passed to the user
+* `SubscriberOptions::historyRequest`: The number of samples a late-joining subscriber will request from a publisher
```

Comment:
It may be obvious but still: 
A subscriber can at most request historyCapacity samples, i.e. if the publisher buffers less samples the subscriber will be out of luck. I am not sure what happens if it tries to request more than the publisher supports. I think they still match but the subscriber gets less samples. This may be surprising for a user and worth noting for this reason.

Reply:
Added.